core-models/src/main/java/com/anytypeio/anytype/core_models/ObjectWrapper.kt

```diff
@@ -128,7 +128,12 @@ sealed class ObjectWrapper {
                 else -> emptyList()
             }
 
-        val targetObjectType: Id? by default
+        val targetObjectType: String?
+            get() = when (val value = map[Relations.TARGET_OBJECT_TYPE]) {
+                is String -> value
```

Comment:
what is string here? `Id` or `Key`? normally, it should be id, and nor unique key. but whatever it is, let's use `typealias`

or you can-reuse inline fun `getValues<Id>()` from `ObjectWrapper`:

`val targetObjectType: Id/Key get() = getValues<Key/Id>(Relations.TARGET_OBJECT_TYPE).firstOrNull
`

Reply:
updated to Id

Reply:
updated to getValues<Id>